README.md

```diff
@@ -97,68 +97,121 @@ Most of the data currently being saved in Bonsai is packaged in a HARP message f
-|                           |        Gust Wind      |      248      |               Timestamped(float)              |
-|                           |        Air Temp       |      249      |               Timestamped(float)              |
-|                           |      XOrientation     |      250      |               Timestamped(float)              |
-|                           |      YOrientation     |      251      |               Timestamped(float)              |
-|                           |        NullValue      |      252      |               Timestamped(float)              |
-|                           |                       |               |                                               |
-|       **PupilLabs**       | WorldCamera (Decoded) |      209      |            Timestamped(HasFrame?)             |
-|                           |    WorldCamera (Raw)  |      210      |            Timestamped(FrameNumber)           |
-|                           |          IMU          |      211      |            Timestamped(FrameNumber)           |
-|                           |         Gaze          |      212      |            Timestamped(FrameNumber)           |
-|                           |         Audio         |      213      |            Timestamped(FrameNumber)           |
-|                           |          Key          |      214      |            Timestamped(FrameNumber)           |
-|       **Omnicept**        |      EyeTracking      |      215      |            Timestamped(long[])                |
-|                           |       HeartRate       |      216      |            Timestamped(long[])                |
-|                           |          IMU          |      217      |            Timestamped(long[])                |
-|                           |         Mouth         |      218      |            Timestamped(long[])                |
-|     **VRTransform**       |       VrTimestamp     |      219      |            Timestamped(long)                  |
-|     **UnityImage**        |       VrTimestamp     |      220      |            Timestamped(long)                  |
+|         **Device**        |      **Stream**       | **Code** |   **Rate**    |                    **Obs**                    |
+|:-------------------------:|:---------------------:|:--------:|:-------------:|:---------------------------------------------:|
+|        **BioData**        |     EnableStreams     |    32    |       -       |  Enable Oximeter, ECG, GSR or Accelerometer   |
+|                           |     DisableStreams    |    33    |       -       |  Enable Oximeter, ECG, GSR or Accelerometer   |
+|                           |          ECG          |    35    |     1 kHz     |  [ECG][1] and Photodiode stream @ 1 kHz (mv)  |
+|                           |          GSR          |    36    |     4 kHz     |  GSR stream @ 4 Hz                            |
```

Comment:
The sampling rate units in the observations and the `Rate` column are inconsistent. Also do we know the units of the GSR sensor?

Reply:
Removed redundant units and will add more information in a separate PR.